README.md

```diff
@@ -102,11 +102,31 @@ dot -Tpdf ParserImpl.dot > ParserImpl.pdf
     ```
     mkdir build
     cd build
-    cmake .. [-DCMAKE_BUILD_TYPE=RELEASE|DEBUG] [-DCMAKE_INSTALL_PREFIX=<path>] [-DENABLE_DOCS=ON (default off)] [-DENABLE_P4RUNTIME_TO_PD=OFF (default on)] [-DENABLE_PROTOBUF_STATIC=OFF (default on)] [-DENABLE_GC=OFF (default on)]
+    cmake .. <optional arguments>
     make -j4
     make -j4 check
     ```
-    If adding new targets to this build system, please see [instructions](#defining-new-cmake-targets).
+    The cmake command takes the following optional arguments to
+    further customize the build:
+     - `-DCMAKE_BUILD_TYPE=RELEASE|DEBUG` -- set CMAKE_BUILD_TYPE to
```

Comment:
Should this be `Release|Debug` or is case unimportant?  The cmake docs are unclear.  Might also want to mention `RelWithDebInfo` to build with optimization and generate debug symbols?